### ISSUE TITLE
DNS root server lookups and zone forwarders

### ISSUE THREAD

**Issue by ajw1980** [xcat2/xcat-core#6401]
I was having long timeouts for unqualified hosts that didn't exist. I figured out that for those hosts, bind was trying to do look ups to the root DNS servers. I think it would probably make sense to have 'forwarders {};' for each zone and set 'forward only;' globally. This configuration should make bind stop trying to go to the root DNS servers:

options {
        directory "/var/named/";
        allow-recursion { any; };
        forwarders {
                10.10.1.1;
                10.10.1.2;
        };
        forward only;
        empty-zones-enable no;
};
zone "26.12.10.IN-ADDR.ARPA." in {
        type master;
        forwarders {};
        allow-update {
                key xcat_key;
        };
        file "db.10.12.26";
};

**Comment by cxhong** [comment]
@ajw1980 , Thanks .  Have u test it out after added `forwarders{}` to each zone?

**Comment by ajw1980** [comment]
Yes, I am currently running with this config for all the zones that xcat creates. I noticed some commands were taking longer than expected, so I made this change. The network I am on does not return immediately for hosts it can't reach, but takes some time to actually timeout.
